arangod/Aql/RestAqlHandler.cpp

```diff
@@ -437,14 +437,20 @@ RestStatus RestAqlHandler::useQuery(std::string const& operation, std::string co
 
   // the PUT verb
   Query* query = nullptr;
-  if (findQuery(idString, query)) {
+  if (!findQuery(idString, query)) {
```

Comment:
was this a bug before?

Reply:
No. The return value was just counter-intuitive, so I swapped it. (https://github.com/arangodb/arangodb/pull/10091/files#diff-c3f6561d1392ef98d98fdcdb2359f25aR603 and https://github.com/arangodb/arangodb/pull/10091/files#diff-c3f6561d1392ef98d98fdcdb2359f25aR598)